desc/objectives/_coils.py

```diff
@@ -1057,13 +1076,30 @@ def build(self, use_jit=True, verbose=1):
 
         plasma_coords = jnp.array([data["R"], data["phi"], data["Z"]]).T
 
+        field = self._field
+
+        if hasattr(field, "Phi_mn"):
+            # make the transform for the FourierCurrentPotentialField
+            if self._field_grid is None:
+                self._field_grid = LinearGrid(
+                    M=30 + 2 * max(field.M, field.M_Phi),
```

Comment:
I guess the 30 is arbitrary and you want the larger of the two resolutions? What's the difference between `field.M` and `field.M_phi`?

Reply:
`field.M` is the `FourierRZToroidalSurface` resolution while `M_Phi` is the resolution of the current potential descriving the surface current, yea we have `30` as a base because even if the field and surface are simple, we need a certain baseline amt of resolution to get an accurate biot savart integral over the source